clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/SerializeApplicationAtomicOperationUnitSpec.groovy

```diff
@@ -170,7 +169,8 @@ class SerializeApplicationAtomicOperationUnitSpec extends Specification {
       def autoscalingMap = [name: SERVER_GROUP_NAME,
                             target: "\${google_compute_instance_group_manager.${SERVER_GROUP_NAME}.self_link}",
                             zone: SERVER_GROUP_ZONE,
-                            autoscaling_policy: autoscalingPolicyMap]
+                            autoscaling_policy: autoscalingPolicyMap,
+                            project: null]
```

Comment:
Filling out project isn't necessary for Terraform but serialization always occurs within the scope of an account so project should always get set by the atomic operation. For the tests I didn't set up an account, so the project is set to null by default. Do you think I should I mock an account for the tests or change the operation to not set project if an account is not specified (which should never actually happen)